From f93776172ec8ae66153adaafd35ec1feeac103e6 Mon Sep 17 00:00:00 2001
From: Mianhan Liu <liumh1@shanghaitech.edu.cn>
Date: Thu, 30 Sep 2021 23:01:22 +0800
Subject: [PATCH] net/atm/atm_misc.c: remove superfluous headers

atm_misc.c hasn't use any macro or function declared in linux/atm.h,
linux/atomic.h and linux/bitops.h.
Thus, these files can be removed from atm_misc.c safely without
affecting the compilation of the ./net module
---
 net/atm/atm_misc.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/net/atm/atm_misc.c b/net/atm/atm_misc.c
index a30b83c1c..bdc69c9f6 100644
--- a/net/atm/atm_misc.c
+++ b/net/atm/atm_misc.c
@@ -4,13 +4,10 @@
 /* Written 1995-2000 by Werner Almesberger, EPFL ICA */
 
 #include <linux/module.h>
-#include <linux/atm.h>
 #include <linux/atmdev.h>
 #include <linux/skbuff.h>
 #include <linux/sonet.h>
-#include <linux/bitops.h>
 #include <linux/errno.h>
-#include <linux/atomic.h>
 
 int atm_charge(struct atm_vcc *vcc, int truesize)
 {
-- 
2.25.1

